### PR TITLE
Brings in payment submodule to demo picker

### PR DESCRIPTION
**Pull request by aristidesstaffieri**
This adds the react payment submodule to the demo picker.
Adds the repo as a submodule.
Exposes it in the demo picker and as a stand alone start command from the root.

Screenshot 2023-06-06 at 3 36 25 PM

### CODE REVIEW

File: demo-picker/src/demo-picker/index.tsx [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import React from "react"
 import { Card, Heading, Layout, Select } from "@stellar/design-system"
 
+import { SendPayment } from "../../../modules/payment-react/src/components/send-payment"
```
Comment (aristidesstaffieri) [inline demo-picker/src/demo-picker/index.tsx:4]: Still not sure why I can't use the `@modules` import syntax here 🤔

Comment (piyalbasu) [reply demo-picker/src/demo-picker/index.tsx:4]: I believe the directory will need to be called `@modules` instead of `modules`

Comment (piyalbasu) [reply demo-picker/src/demo-picker/index.tsx:4]: And in the payment-react repo, the package name will need to be changed to @modules/payment-react

Comment (aristidesstaffieri) [reply demo-picker/src/demo-picker/index.tsx]: fixed in [be41f28](https://github.com/stellar/scaffold-soroban/pull/3/commits/be41f28509a110975df1a6a7455efafb236089a8)

**Review by piyalbasu — APPROVED** [review]

**Pull request merged**